aloha/real_env.py

```diff
@@ -136,33 +127,56 @@ def setup_robots(self):
         setup_follower_bot(self.follower_bot_right)
 
     def get_qpos(self):
```

Comment:
Do we want to go through the effort of adding return typing and docstrings to each method?